composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.5.9 <7.0",
```

Comment:
I think Cachet works on 7.0 but not 7.1+. I can't remember which version had the `count` changes.

Reply:
7.2 I think?